forge-gui/src/main/java/forge/itemmanager/GroupDef.java

```diff
@@ -106,7 +106,7 @@ public Integer apply(final InventoryItem item) {
             }),
 
     CARD_TYPE("lblType",
-            new String[] { "Planeswalker", "Creature", "Sorcery", "Instant", "Artifact", "Enchantment", "Land", "Tribal instant", "Battle" },
+            new String[] { "Planeswalker", "Creature", "Sorcery", "Instant", "Artifact", "Enchantment", "Land", "Kindred instant", "Battle" },
```

Comment:
Though I'd like to be sure that `"Kindred instant"` is right. Since, at first glance, shouldn't it be just `"Kindred"`?

Reply:
Removed the category, assuming no word to the contrary at this point is a "go ahead".